__tests__/disabling-tokenizers.test.js

```diff
@@ -0,0 +1,22 @@
+const markdown = require('../index');
+
+test('disabling inlineCode', () => {
+  const md = '`const js = true `';
+  const opts = { disableTokenizers: { inline: ['code'] } };
```

Comment:
Ahhh, I see. I'd prefer if we could just pass a simple boolean in to the option. Does the user really need such fine grained control over what is and isn't considered inline? Can we make that decision for them?

Reply:
Well, the driving force is when the editor deserializes markdown, it doesn't parse inlines and gets back plain text. Then it parses that plain text with just the inlines enabled to do the decoration.

Right now, my plan is to disable all inlines except `links` during deserialization. From a development standpoint, I'd like to be able to twiddle the tokenizers while I figure things out.

Reply:
As per slack, I'll update this with simpler options.